.github/workflows/PR-check.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: PR Check
+
+on: 
+#  pull_request:
+#    paths-ignore:
+#      - '.github/workflows/**'
+#      - 'test-script'  
+  workflow_dispatch:
+
+env:
+  ARTIFACT_VERSION: dummy-verison 
+
+jobs:
+  build:
+    name: PR Check
```

Comment:
By default, the status check required for a PR under [rax-maas management](https://github.com/rax-maas/rax-maas-github-settings) is named "test".